types/drupal-attribute/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
```

Comment:
Based on the export, it looks like `esModuleInterop` should be removed and the test should use `import DrupalAttribute = require('drupal-attribute');`.

Reply:
Done.